src/styles/base.scss

```diff
@@ -219,9 +225,10 @@ ol:not([class]) {
     white-space: nowrap;
     width: 1px;
   }
+
   [data-amplify-theme='default-theme'] & {
     &:focus-visible {
       outline-color: var(--amplify-colors-white);
     }
   }
-}
+}
```

Comment:
Should this file be reset?

Also, can you take a look at your editor settings and see if there is some reason final new lines aren't being added? I'm a little surprised prettier isn't picking this up on commit (I think it should by default?) but it should also be something you can turn on/configure so these don't pop up in a lot of your changed files.